packages/x-charts/src/internals/material/augmentation.ts

```diff
@@ -0,0 +1,37 @@
+import type { ButtonProps } from '@mui/material/Button';
+import type { IconButtonProps } from '@mui/material/IconButton';
+import type { DividerProps } from '@mui/material/Divider';
+import type { MenuItemProps } from '@mui/material/MenuItem';
+import type { MenuListProps } from '@mui/material/MenuList';
+import type { PopperProps } from '@mui/material/Popper';
+import type { TooltipProps } from '@mui/material/Tooltip';
+
+declare module '@mui/x-charts' {
+  interface ChartBaseButtonPropsOverrides {
+    material?: Partial<ButtonProps>;
+  }
+
+  interface ChartBaseIconButtonPropsOverrides {
+    material?: Partial<IconButtonProps>;
+  }
+
+  interface ChartBaseDividerPropsOverrides {
+    material?: Partial<DividerProps>;
+  }
+
+  interface ChartBaseMenuItemPropsOverrides {
+    material?: Partial<MenuItemProps>;
+  }
+
+  interface ChartBaseMenuListPropsOverrides {
+    material?: Partial<MenuListProps>;
+  }
+
+  interface ChartBasePopperPropsOverrides {
+    material?: Partial<PopperProps>;
+  }
+
+  interface ChartBaseTooltipPropsOverrides {
+    material?: Partial<TooltipProps>;
+  }
+}
```

Comment:
Why does this exist? Why is it not in the overrides definition?

Reply:
[It's a copy of the way the Data Grid does it](https://github.com/mui/mui-x/blob/7966bf543079e9b62d8d95046a005481feb7b1f3/packages/x-data-grid/src/material/augmentation.ts#L21). 

I suppose the idea is that eventually users will be able to choose between Material UI and other options, so we don't want to define it directly in the props.